src/LazyImage/src/DependencyInjection/LazyImageExtension.php

```diff
@@ -46,7 +46,11 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container
             ->setDefinition('lazy_image.blur_hash', new Definition(BlurHash::class))
-            ->setArgument(0, new Reference('lazy_image.image_manager', ContainerInterface::NULL_ON_INVALID_REFERENCE))
+            ->setArguments([
+                new Reference('lazy_image.image_manager', ContainerInterface::NULL_ON_INVALID_REFERENCE),
+                null, // $cache
+                null, // $fetchImageContent
+            ])
```

Comment:
Looks like I can't set the 3rd argument if the 2nd argument has not been set, which can be the case if we have no cache configured, that's why I've changed to `->setArguments`